h/static/styles/partials-v2/_search.scss

```diff
@@ -0,0 +1,128 @@
+// Search page
```

Comment:
Styles apply to components rather than pages, the difference being that a UI component might be re-used across multiple pages. For example, we'll probably have views in future that show single annotations or all annotations from a document and these would use the same components, though they may or may not be served at the `/search` URL.

So I would suggest here that the description should be something like "Annotation search result list".